src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -97,21 +90,10 @@ export const NavBar: React.FC<NavBarPropsType> = ({
 
   const shouldShowHomeButton = hasSovereignVenue && !isSovereignVenue;
 
-  const {
-    hasSelectedProfile,
-    openUserProfileModal,
-    updateUserProfileData,
-    selectedUserProfile,
-  } = useProfileModalControls();
-
-  const isSameUser = useIsCurrentUser(selectedUserProfile?.id);
-
-  useEffect(() => {
-    if (hasSelectedProfile && isSameUser) updateUserProfileData(userWithId);
-  }, [hasSelectedProfile, isSameUser, updateUserProfileData, userWithId]);
```

Comment:
That's a "hack" that was needed to properly update profile after saving. Not needed anymore.